backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/order/createFrom/po_from_so/impl/CreatePOFromSOsAggregator.java

```diff
@@ -332,6 +332,7 @@ private I_C_Order createPurchaseOrder(
 		purchaseOrder.setUser1_ID(salesOrder.getUser1_ID());
 		purchaseOrder.setUser2_ID(salesOrder.getUser2_ID());
 		purchaseOrder.setC_Currency_ID(salesOrder.getC_Currency_ID());
+		purchaseOrder.setDatePromised(salesOrder.getDatePromised());
```

Comment:
if this always wanted though? even if `GroupLinesByPromisedDate`=N ?

Reply:
Not sure, so better safe than sorry.